### PR TITLE
Feature/add log

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: test/acceptance/README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Its needed:
   "environment": {
     "name": "orion",
     "logs_path": "logs" # Path where the logs will be stored
+    "log_level": "ERROR" # Level of the logs inside the tests
```
Comment (iariasleon) [inline test/acceptance/README.md]: recommend to append the allowed values in log_level. ex: ERROR, INFO, etc

File: test/acceptance/integration/tools/general_utils.py [hunk 2]
```diff
@@ -332,6 +341,7 @@ def stop_cb():
             if get_cb_pid() != '':
                 raise EnvironmentError('After try to kill the Context Broker process, is still running, kill it manually')
     world.cb_pid = get_cb_pid() # It should be '' (empty)
+    return True
```
Comment (iariasleon) [inline test/acceptance/integration/tools/general_utils.py]: Is it neccesary this return? It always returns True

Comment (jcalderin) [inline test/acceptance/integration/tools/general_utils.py]: Fixed in 5f4d0d9

File: test/acceptance/README.md (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/README.md]: Fixed in 5f4d0d9

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**